product-mini/platforms/posix/main.c

```diff
@@ -67,6 +66,9 @@ print_help()
     printf("                           Use comma to separate, e.g. --enable-segue=i32.load,i64.store\n");
     printf("                           and --enable-segue means all flags are added.\n");
 #endif
+#endif /* WASM_ENABLE_JIT != 0*/
+#if WASM_ENABLE_AOT != 0 || WASM_ENABLE_JIT != 0
+    printf("  --perf-profile           Enable linux perf support. It works in aot and llvm-jit.\n");
```

Comment:
Is it better to use `--enable-perf-profile`?

Reply:
how about `--support-perf` or `--enable-perf`? Don't want to make it longer